src/Commands/play.js

```diff
@@ -0,0 +1,88 @@
+// In JavaScript, classes are closures. Thus, setupGame 
+// can be seen as a class. 
+const setupGame = async function(client, message, args) {
+  if (typeof this.channelList === "undefined") {
+    this.channelList = [];
+  }
+  
+  if (this.channelList.includes(message.channel.id)) {
+    message.channel.send("This channel is taken nya!");
+    return;
+  }
+  this.channelList.push(message.channel.id);
+  
+  let currentValue = 0;
+  let messageId = null;
+  let currentChannel = message.channel.id;
+  message.channel.send(currentValue).then((sent) => {
+    messageId = sent.id;
+    runGameLogic(client, sent);
+  });
+  
+  // Here, this refers to the setupGame class
+  const refreshMessage = (client, message) => {
+    message.edit(currentValue);
+  }
+
+  const updateGameMessage = function(messageReaction, user) {
+    if (!user.bot) {
+      if (messageId == messageReaction.message.id) {
+        let hasStopped = false;
+        switch (messageReaction.emoji.toString()) {
+          case "⬅️":
+            currentValue--;
+            break;
+          case "➡️":
+            currentValue++;
+            break;
+          case "❌":
+            messageReaction.message.reactions.removeAll();
+            hasStopped = true;
+            this.emit("stop", messageReaction.message.channel.id);
+            break;
+          default:
+            break;
+        }
+        messageReaction.users.remove(user);
+        if (!hasStopped) {
+          refreshMessage(this, messageReaction.message);
+        }
+      }
+    }
+  }
+
+  const runGameLogic = (client, message) => {
+    message.react("⬅️");
+    message.react("➡️");
+    message.react("❌");
+    /*
+    let activeMessageID = client.variables.gameLogic.currentMessage;
+    message.channel.messages.fetch(activeMessageID)
+      .then(message => {
+      
+    })*/
+  }
+  
+  client.on("messageReactionAdd", updateGameMessage);
+  client.on("stop", (channelId) => {
+    if (channelId === currentChannel) {
+      this.channelList.splice(this.channelList.indexOf(channelId), 1);
+      client.removeListener("messageReactionAdd", updateGameMessage);
+      client.removeListener("stop", arguments.callee);
```

Comment:
arguments.callee is not allowed in "strict" mode, but at the same time I want a way to decommission the entire module when a stop signal is fired. How should we do this?